src/FontManager.ts

```diff
@@ -41,11 +41,18 @@ export type FontManagerOptions = {
   fontSize?: number;
 };
 
+type EventMap = {
+  atlasChange: object;
+  error: { error: Error };
+};
+
+const MAX_CHARS = 2048;
```

Comment:
What's the reason for this limit? A comment here would be useful. If it depends on platform or external factors maybe it's worth passing as a constructor parameter to the `FontManager`?

Reply:
I could probably remove this... I think I was going for trying to add a soft limit where the performance starts degrading rather than waiting until completely running out of space in the atlas. But not really required.

Reply:
yeah maybe remove then